ConsumerApi.Tests.Integration/Support/Constants.cs

```diff
@@ -2,5 +2,13 @@
 
 public static class Constants
 {
+    public const string CLIENT_ID = "test";
+    public const string CLIENT_SECRET = "test";
```

Comment:
You should use the configured client credentials instead (see DevicesStepDefinitions line 39)

Reply:
Changes made.